### PR TITLE
Dataset: TabLib

### PR DESCRIPTION
**Pull request by jpgard**
This PR is submitted in accordance with the Hyak dataset policy described [here](https://hyak.uw.edu/blog/hyak-dataset-policy/).

### CODE REVIEW

File: docs/data-commons/tablib.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+)
+
+df = ds['train'].to_pandas()
+
+tables = [pa.RecordBatchStreamReader(b).read_all() for b in df['arrow_bytes']]
+
+```
+# How to access?
+
+Users who access the data should also apply for public, open credentialized access to the dataset on Hugging Face DAtasets [here](https://huggingface.co/datasets/approximatelabs/tablib-v1-full).
```
Comment (pnw-ryanmcgr) [inline docs/data-commons/tablib.md]: DAtasets > Datasets

**Review by pnw-ryanmcgr — COMMENTED** [review]
Otherwise, looks great to me.

File: docs/data-commons/tablib.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
+Sponsoring groups are Ludwig Schmidt, Tim Althoff, and Pang Wei Koh. Student users are Josh Gardner, Mike Merrill, and Vinayak Gupta.
```
Comment (pnw-ryanmcgr) [inline docs/data-commons/tablib.md:6]: Also, you will need to add the following block to the very top of the document
```
---
id: tablib
title: TabLib
---
```

File: docs/data-commons/tablib.md (hunk 1 above)
Comment (jpgard) [reply docs/data-commons/tablib.md]: done.

File: docs/data-commons/tablib.md (hunk 2 above)
Comment (jpgard) [reply docs/data-commons/tablib.md:6]: done.

**Comment by jpgard** [comment]
Thanks for the review @pnw-ryanmcgr !

Let me know what the next steps are, once we have the right levels of approval, so that we can provide any support in getting the actual data hosted!

**Review by pnw-ryanmcgr — APPROVED** [review]

**Pull request merged**